Doc/library/stdtypes.rst

```diff
@@ -4844,19 +4848,22 @@ mistakes like ``dict[str][str]``::
      File "<stdin>", line 1, in <module>
    TypeError: There are no type variables left in dict[str]
 
-However, that syntax is valid when type variables are used::
+However, that expression is valid when :ref:`type variables <generics>` are
```

Comment:
Well, _that particular expression_ will never be valid, but other such expressions could be.
```suggestion
However, such expressions are valid when :ref:`type variables <generics>` are
```